rust/processor/src/models/mod.rs

```diff
@@ -16,3 +18,8 @@ pub mod token_models;
 pub mod token_v2_models;
 pub mod transaction_metadata_model;
 pub mod user_transactions_models;
+
```

Comment:
if v1 name is "XXXEvent" then v2 name is always "XXX" or "XXX<YYY..." and emitted just before v1 so if the preceding event exists and matches the name we will dedup/skip the current v1.

Reply:
nit: can you add this as a comment in the code, and make a note that it's for events v2 migration?